app/features-json/view_models/build_view_model.rb

```diff
@@ -0,0 +1,76 @@
+require_relative "../../shared/models/build"
+require_relative "../../shared/json_convertible"
+
+module FastlaneCI
+  # View model to expose the detailed info about a build.
+  class BuildViewModel
+    include FastlaneCI::JSONConvertible
+
+    # The project ID this build is associated with
+    attr_reader :project_id
+
+    # @return [Integer]
+    attr_reader :number
+
+    # @return [String]
+    attr_reader :status
+
+    # @return [DateTime] Start time
+    attr_reader :timestamp
+
+    # @return [Integer]
+    attr_reader :duration
+
+    # @return [String] An optional message to go along with the build, will show up as part of the build status on
+    # GitHub
+    attr_reader :description
+
+    # @return [String] the trigger type that triggered this particular build
+    attr_reader :trigger
+
+    # @return [String] the lane name (without platform) that was used for this particular build
+    attr_reader :lane
+
+    # @return [String] the platform name (without lane) that was used for this particular build
+    attr_reader :platform
+
+    # @return [Hash] the parameters that were passed on this particular build
+    # TODO: We currently don't use/store/support parameters (yet) https://github.com/fastlane/ci/issues/783
+    attr_reader :parameters
+
+    # @return [Hash] a hash containing the version numbers for each build tool that was used
+    attr_reader :build_tools
+
+    # @return [String] The git URL
+    attr_reader :clone_url
+
+    # @return [String] The git branch
+    attr_reader :branch
+
+    # @return [String] The git ref
+    attr_reader :ref
```

Comment:
@KrauseFx what's the git ref?

Reply:
We have some more docs on this over here https://github.com/fastlane/ci/blob/master/app/shared/models/git_fork_config.rb#L19-L21

I don't know for sure when this gets filled out, as every `ref` can also be represented by a `sha`. Maybe @taquitos knows more, as he invented the class `GitForkConfig`